sync_labels.py

```diff
@@ -0,0 +1,207 @@
+        print_list(list(dst_add), " [+] ")
+
+    if len(dst_upd) > 0:
+        logger.info(f"The following labels will be updated in {dst_repo}:")
+        print_list(list(dst_upd), " [u] ")
+
+    if delete:
+        if len(dst_del) > 0:
+            logger.info(f"The following labels will be deleted in {dst_repo}:")
+            print_list(list(dst_del), " [-] ")
+
+    # Add new labels
+    add_ok = []
+    add_fail = []
+    for name in dst_add:
+        if dry_run:
+            add_ok.append(name)
+        else:
+            print_sameline("Adding label: " + name)
+            label = src.get_label(name)
+            color = label.color
+            description = label.description
+            try:
+                dst.create_label(name, color, description)
+            except GithubException:
+                logging.exception(f"Error adding label: {name}")
+                add_fail.append(name)
+            else:
+                add_ok.append(name)
+
+    # Update existing labels
+    upd_ok = []
+    upd_fail = []
+    for name in dst_upd:
+        if dry_run:
+            upd_ok.append(name)
+        else:
+            print_sameline("Updating label: " + name)
+            label = src.get_label(name)
+            dst_label = dst.get_label(name)
+            color = label.color
+            description = label.description
+            try:
+                dst_label.edit(name, color, description)
+            except GithubException:
+                logging.exception(f"Error updating label: {name}")
+                upd_fail.append(name)
+            else:
+                upd_ok.append(name)
+
+    # Delete labels
+    del_ok = []
+    del_fail = []
+    if delete:
+        for name in dst_del:
+            if dry_run:
+                del_ok.append(name)
+            else:
+                print_sameline("Deleting label: " + name)
+                label = dst.get_label(name)
+                try:
+                    label.delete()
+                except GithubException:
+                    logging.exception(f"Error deleting label: {name}")
+                    del_fail.append(name)
+                else:
+                    del_ok.append(name)
+
+    if not dry_run:
+        print_sameline("Done.", last_line=True)
+
+    if len(add_fail) > 0:
+        logger.info(f"Failed to add labels to {src_repo}:")
+        print_list(add_fail, " [!] ")
+    if len(upd_fail) > 0:
+        logger.info(f"Failed to update labels in {src_repo}:")
+        print_list(add_fail, " [!] ")
+    if len(del_fail) > 0:
+        logger.info(f"Failed to delete labels from {src_repo}:")
+        print_list(del_fail, " [!] ")
+
+
+def print_sameline(msg, last_line=False):
+    sys.stdout.write("\033[K") # Clear prior printed line
+    if not last_line:
+        print(msg, end="\r")
+    else:
+        print(msg)
+
+
+def get_access_token():
+    """Grab Github API token from environment variable"""
+    api_key = os.environ.get("GITHUB_API_KEY", None)
```

Comment:
```suggestion
    api_key = os.environ.get("GITHUB_API_KEY")
```

but really this should using `try: … except KeyError:`

Reply:
opted for try: except.